etl/steps/data/garden/explorers/2021/food_explorer.ipynb

```diff
@@ -2939,24 +3138,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 56,
+   "execution_count": 262,
    "id": "0042dcdd-2cc2-460b-9523-e3167bcaef2b",
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "1.5M\t/tmp/food_explorer/vegetables.csv\n",
-      "1.5M\t/tmp/food_explorer/oilcrops.csv\n",
-      "1.5M\t/tmp/food_explorer/fruit.csv\n",
-      "1.4M\t/tmp/food_explorer/wheat.csv\n",
-      "1.4M\t/tmp/food_explorer/pulses.csv\n",
-      "1.4M\t/tmp/food_explorer/milk.csv\n",
-      "1.4M\t/tmp/food_explorer/meat_poultry.csv\n",
-      "1.4M\t/tmp/food_explorer/maize.csv\n",
-      "1.3M\t/tmp/food_explorer/tomatoes.csv\n",
-      "1.3M\t/tmp/food_explorer/rice.csv\n"
+      "1.1M\t/tmp/food_explorer/vegetables.csv\n",
```

Comment:
wait where are these size decreases coming from? are we dropping rows because we don't have a population value for them?

Reply:
That is correct. Mainly dropping data for former countries: 

'Serbia and Montenegro', 'Macao', 'USSR', 'Niue', 'French Polynesia', 'Tokelau', 'Melanesia', 'Bermuda', 'Czechoslovakia', 'Yugoslavia', 'Polynesia', 'Cook Islands', 'Eritrea and Ethiopia'.

I think there is work to do to add missing population numbers for some countries. This can be a follow-up mini project (https://github.com/owid/etl/blob/master/etl/steps/data/garden/owid/latest/key_indicators/table_population.py)